examples/mms/tts/tutorial/MMS_TTS_Inference_Colab.ipynb

```diff
@@ -361,6 +361,15 @@
         "from models import SynthesizerTrn\n",
         "from scipy.io.wavfile import write\n",
         "\n",
+        "def preprocess_char(text, lang=None):\n",
+        "    \"\"\"\n",
+        "    Special treatement of characters in certain languages\n",
+        "    \"\"\"\n",
+        "    print(lang)\n",
+        "    if lang == 'ron':\n",
```

Comment:
This looks good for now. But, if we end up having too many custom rules - it might be better to download the tts infer file in the colab notebook and use it directly so that we have only one source of truth. 
`wget https://github.com/facebookresearch/fairseq/blob/main/examples/mms/tts/infer.py`.